tests/ui/traits/next-solver/opaques/stranded_opaque.rs

```diff
@@ -0,0 +1,33 @@
+//@ compile-flags: -Znext-solver
+
+// Test for https://github.com/rust-lang/trait-system-refactor-initiative/issues/235
+
+// These are cases where an opaque types become "stranded" due to
+// some errors. Make sure we don't ICE in either case.
+
+// Case 1: `impl Send` is stranded
```

Comment:
can u also make equivalent tests for all these cases using a TAIT instead of an RPIT. I don't think they were ICEing but good to test anyway